tools/asciidoc_to_fragments.py

```diff
@@ -64,6 +64,19 @@ def get_event_timestamp(repository, event, number):
         date = datetime.fromisoformat(data["closed_at"].replace('Z', '+00:00'))
         return str(int(datetime.timestamp(date)))
 
+
+def sanitize_title(title):
```

Comment:
I'm doing the same in a separate PR https://github.com/elastic/elastic-agent-changelog-tool/pull/104 so I would not include this here

Reply:
Please use it there if useful, will remove it now.